File: ProgrammingFundamentals/C#/03.DataTypesAndVariables/Exercises/P09_ReverseCharacters/P09_ReverseCharacters.cs
using System;

namespace P09_ReverseCharacters
{
    class P09_ReverseCharacters
    {
        static void Main(string[] args)
        {
            char first = char.Parse(Console.ReadLine());
            char second = char.Parse(Console.ReadLine());
            char third = char.Parse(Console.ReadLine());

            Console.WriteLine("" + third + second + first);
        }
    }
}

File: ProgrammingFundamentals/Java/03.DataTypesAndVariables/Exercises/src/P09_ReverseCharacters.java
import java.util.Scanner;
public class P09_ReverseCharacters {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);

        char first = console.nextLine().charAt(0);
        char second = console.nextLine().charAt(0);
        char third = console.nextLine().charAt(0);

        System.out.println("" + third + second + first);
    }
}
